common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4773,7 +4778,24 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     HIVE_QUERY_REEXECUTION_STATS_CACHE_SIZE("hive.query.reexecution.stats.cache.size", 100_000,
         "Size of the runtime statistics cache. Unit is: OperatorStat entry; a query plan consist ~100."),
     HIVE_QUERY_PLANMAPPER_LINK_RELNODES("hive.query.planmapper.link.relnodes", true,
-        "Wether to link Calcite nodes to runtime statistics."),
+        "Whether to link Calcite nodes to runtime statistics."),
+
+    HIVE_SCHEDULED_QUERIES_EXECUTOR_ENABLED("hive.scheduled.queries.executor.enabled", true,
+        "Controls whether HS2 will run scheduled query executor."),
+    HIVE_SCHEDULED_QUERIES_NAMESPACE("hive.scheduled.queries.namespace", "hive",
+        "Sets the scheduled query namespace to be used. New scheduled queries are created in this namespace;"
+            + "and execution is also bound to the namespace"),
+    HIVE_SCHEDULED_QUERIES_EXECUTOR_IDLE_SLEEP_TIME("hive.scheduled.queries.executor.idle.sleep.time", "60s",
+        new TimeValidator(TimeUnit.SECONDS),
+        "Time to sleep between quering for the presence of a scheduled query."),
+    HIVE_SCHEDULED_QUERIES_EXECUTOR_PROGRESS_REPORT_INTERVAL("hive.scheduled.queries.executor.progress.report.interval",
+        "60s",
+        new TimeValidator(TimeUnit.SECONDS),
+        "While scheduled queries are in flight; "
+            + "a background update happens periodically to report the actual state of the query"),
+    HIVE_SECURITY_AUTHORIZATION_SCHEDULED_QUERIES_SUPPORTED("hive.security.authorization.scheduled.queries.supported",
```

Comment:
Is this necessary? I am not sure how this code is working, but I guess I am wondering why do we need specifically this for schedules queries?